test/units/plugins/inventory/test_inventory.py

```diff
@@ -156,6 +159,26 @@ def test_ini_variables_stringify(self):
             else:
                 self.assertIsInstance(variables['var%s' % i], type(values[i]))
 
+    @mock.patch('ansible.inventory.manager.unfrackpath', mock_unfrackpath_noop)
+    @mock.patch('os.path.exists')
+    @mock.patch('os.access')
+    def test_yaml_inventory(self, p_access, p_exist):
+        inventory_content = """
+---
```

Comment:
[textwrap.dedent](https://docs.python.org/2.7/library/textwrap.html?#textwrap.dedent) could be used here.